src/compiler/crystal/codegen/codegen.cr

```diff
@@ -5,7 +5,7 @@ require "../semantic/type_inference"
 require "../program"
 require "./llvm_builder_helper"
 
-LLVM.init_x86
+LLVM.init_all_targets
```

Comment:
I never submitted that patch because it was quite slow for me, adding a very noticeable lag to compiling anything.